packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -131,7 +131,7 @@ export class CreateAccountStore {
     const words = phrase.split(' ');
     const PARITY_WORDLIST = getParityWordlist();
     if (
-      words.length < 12 ||
+      words.length < 11 ||
```

Comment:
could we use a constant for this instead of a magic number?
i.e. `const MAX_PARITY_SIGNER_RECOVERY_WORDS = 11;`